src/router/index.js

```diff
@@ -100,7 +105,7 @@ The code below works on `isAuthenticated` state and before every route:
 2. Redirects user to home if user is already logged in and visiting a page that is intended for guest (route.meta.guest)
 */
 
-router.beforeEach((to, from, next) => {
+router.beforeEach((to, from) => {
```

Comment:
why remove this?

Reply:
https://next.router.vuejs.org/guide/advanced/navigation-guards.html#optional-third-argument-next